beacon-chain/state/stateutil/benchmark_test.go

```diff
@@ -45,17 +45,11 @@ func BenchmarkBlockHTR(b *testing.B) {
 	})
 }
 
-func BenchmarkMerkleize(b *testing.B) {
+func BenchmarkMerkleize_Buffered(b *testing.B) {
```

Comment:
Updating deps with go mod led to some breaking changes in proto's zszz that are no longer compatible with our old code, so had to amend this function